source/ceylon/math/whole/Whole.ceylon

```diff
@@ -27,11 +27,27 @@ shared interface Whole
                                positive exponent")
     shared formal actual Whole power(Whole exponent);
 
-    "The result of `(this**exponent) % modulus`."
+    deprecated("Renamed to [[modPower]].")
```

Comment:
The method for `%` on Integer is called `remainder`, so `powerRemainder` was named to be consistent with that. I don't think renaming is especially helpful. Considering the recent discussions about `Float.magnitude` vs `absoluteValue`, I'm inclined to think we should stick to the existing name.